### DISCUSSION TITLE
Sebi Boga

### DISCUSSION THREAD

**Discussion by Mede375** [peviitor-ro/community#84] [Scrapers Teams]
1. Medeea Boamfa
2. Oana s
3. Daniel Andrei 
4. Daniela Stanculesc

**Comment by sebiboga** [comment]
@Mede375 avem pe AROBS aici: https://github.com/peviitor-ro/scraper_JMeter/blob/main/JMeter_scraper_arobs.jmx

**Comment by sebiboga** [comment]
@Mede375 trebuie sa ne gasim un NUME DE ECHIPA
ce zici de "jmecker" ? // -- de la JMeter

**Comment by sebiboga** [comment]
@Mede375 avem pe BORGDesign https://github.com/peviitor-ro/scraper_JMeter/blob/main/JMeter_scraper_borgdesign.jmx

**Comment by sebiboga** [comment]
@Medeea735 avem pe BOSCH https://github.com/peviitor-ro/scraper_JMeter/blob/main/JMeter_scraper_bosch.jmx
